### ISSUE TITLE
Solution of 3+ pods on test cluster takes unexpectedly long time

### ISSUE THREAD

**Issue by grandrew** [criticalhop/kalc#164]
On some previous commits, it was significantly faster

Things to test:

- [ ] try alternating heuristic algorithm instead of pure FF
- [ ] try other options for pure FF translation and search

**Comment by grandrew** [comment]
🔝 for tesg1 cluster: 30 pods, 5 nodes

```
mark_3_pods_of_deployment_as_not_at_same_node__poodle_branch__br44-0: 165000
mark_checked_pod_as_antiaffinity_checked_for_target_pod__poodle_branch__br5: 157500
set_antiaffinity_between_3_pods_of_deployment__poodle_branch__br377-0: 95040
moverunningpodtoanothernode__poodle_branch__br1-16: 23940
```

**Comment by grandrew** [comment]
Issue is that `mark_checked_pod_as_antiaffinity_checked_for_target_pod__poodle_branch__br5` need splitter

**Comment by grandrew** [comment]
TOP after removing *3pods:

```
mark_checked_pod_as_antiaffinity_checked_for_target_pod__poodle_branch__br5: 157500
moverunningpodtoanothernode__poodle_branch__br1-16: 23940
mark_2_pods_of_deployment_as_not_at_same_node__poodle_branch__br3-0: 11000
set_antiaffinity_between_2_pods_of_deployment__poodle_branch__br5-0: 5280
moverunningpodtoanothernode__poodle_branch__br1-8 : 3990
```

**Comment by grandrew** [comment]
TOP after splitting with amount of math per action >0 (tesg1):

```
mark_checked_pod_as_antiaffinity_checked_for_target_pod__poodle_branch__br5-0: 77550
moverunningpodtoanothernode__poodle_branch__br1-16: 23940
mark_2_pods_of_deployment_as_not_at_same_node__poodle_branch__br3-0: 11000
set_antiaffinity_between_2_pods_of_deployment__poodle_branch__br5-0: 5280
mark_checked_pod_as_antiaffinity_checked_for_target_pod__poodle_branch__br5-1: 5250
```

**Comment by grandrew** [comment]
With all latest fixes, takes about 1 minute for first result

**Comment by grandrew** [comment]
3+ pods still takes too long

**Comment by grandrew** [comment]
There seems to be some kind of bug that prevents it from solving

Steps to reproduce:

- take tesg1 data
- launch kalc-optimize
- wait for 3 pods run... never returns solution

**Comment by grandrew** [comment]
Old fix with `splitMe > 0` is on 915c374c76a745fd9d834c65081d3910c11ea65e

New problem is with actual search, not translate. Closing
